torch_xla/csrc/aten_xla_type.cpp

```diff
@@ -3502,6 +3502,19 @@ at::Tensor& AtenXlaType::zero_(at::Tensor& self) {
   return self;
 }
 
+at::Scalar AtenXlaType::_local_scalar_dense(const at::Tensor & self) {
+  static bool sync =
+        xla::sys_util::GetEnvBool("XLA_SYNC_BEFORE_ITEM_CALL", true);
+  if (sync) {
+      XLA_FN_COUNTER("xla::");
```

Comment:
Maybe instead of `XLA_FN_COUNTER`, we can use `XLA_COUNTER ` and give it a more explicit name. Something along the line of `EarlySyncTensorCount`